### DISCUSSION TITLE
[BUG] Asset unique y channel bound, no puedo eliminar tareas.

### DISCUSSION THREAD

**Discussion by AlejandroLoraG** [Cotalker/documentation#290] [4. Bug Reports] [answered]
_**Hola!**_

Tengo un workflow que tiene Chat Channels Unbound y Asset tipo Unique. 
El problema es cuando voy a la task view para eliminar _**TASKS PADRES**_ , el sistema no me lo permite. (Este workflow tiene un workflow hijo). 

El problema ocurre en 2 situaciones:

1. **Al mover una task padre sin hijas hacia un estado no ligado al workflow hijo, en este caso, desde el estado 3 en adelante no puedo eliminar una task padre.**

2. **Al mover una task padre con hija(s), en este caso, no puedo eliminar la task padre en ningun otro estado que no sea el inicial.**

Workflows:
![image](https://user-images.githubusercontent.com/51095635/123142741-c0e55480-d427-11eb-80ec-412f60b9516c.png)

Workflow Padre:
![image](https://user-images.githubusercontent.com/51095635/123142839-e07c7d00-d427-11eb-922b-5a05a06f9338.png)

Workflow Hijo:
![image](https://user-images.githubusercontent.com/51095635/123142917-f9852e00-d427-11eb-9c8e-4bdeb0b0e170.png)

Error cuando intento eliminar una task padre:
![image](https://user-images.githubusercontent.com/51095635/123142508-824f9a00-d427-11eb-8510-2b970be4f293.png)

Error por consola debugger de chrome:
![image](https://user-images.githubusercontent.com/51095635/123157271-512b9580-d438-11eb-90ad-df45e554384a.png)

**_Muchas gracias!_**

**Comment by guillevalin** [comment, marked as answer]
@AlejandroLoraG hemos identificado un bug y será subida una corrección que corrija esto.
